hs_core/management/commands/compute_bag_checksum_for_published_resources.py

```diff
@@ -0,0 +1,25 @@
+
+from django.core.management.base import BaseCommand
+
+from hs_core.models import BaseResource
+from hs_core.hydroshare.utils import get_resource_by_shortkey
+from hs_core.tasks import create_bag_by_irods
+
+
+class Command(BaseCommand):
+    help = "Generate bag checksums for published resources as needed"
+
+    def add_arguments(self, parser):
+
+        # a list of resource id's: none does nothing.
```

Comment:
Looks like not passing resource ids runs it for every resource.

Reply:
Good catch. Will update comment.